### ISSUE TITLE
Notification bell does not light up for new users until after the first time they click it?

### ISSUE THREAD

**Issue by gwillen** [LessWrong2/Lesswrong2#613]
EDIT: Originally, I had said that it did not work for me, user 'gwillen', but then they started working. My new theory is that it does not work for new users, and the user 'gwillen-test' is currently in this state if anybody needs an example to investigate. (The 'gwillen-test' account has no private information associated with it, and you may abuse it freely in any way you desire in order to try to debug this.)

@Raemon asked me to file this, after we figured out that the reason I thought notifications weren't a thing yet is that the bell icon never lights up for me, or otherwise indicates that I have any.

I am logged-in, on Chrome 67.0.3396.62, on OS X 10.12.6. I am running uBlock Origin 1.16.8 (adblock), and a few other extensions that really shouldn't be fucking it up but you know how these things are.

A couple of not-relevant-looking errors on console:

"""
p.typekit.net/p.gif?s=1&k=jvr1gjm&ht=tk&h=www.lesswrong.com&f=8498.8499.8500.8501.8502.8503.14739.14740.10954.13453.13454.13455.13456.13457.13458.13459.13460.13461&a=10927367&js=1.19.2&app=typekit&e=js&_=1527972246030:1 GET https://p.typekit.net/p.gif?s=1&k=jvr1gjm&ht=tk&h=www.lesswrong.com&f=8498.8499.8500.8501.8502.8503.14739.14740.10954.13453.13454.13455.13456.13457.13458.13459.13460.13461&a=10927367&js=1.19.2&app=typekit&e=js&_=1527972246030 0 ()
"""

"""
Mixed Content: The page at 'https://www.lesswrong.com/' was loaded over HTTPS, but requested an insecure favicon 'http://res.cloudinary.com/lesswrong-2-0/image/upload/v1497915096/favicon_lncumn.ico'. This request has been blocked; the content must be served over HTTPS.
"""

(Indeed, the favicon is not loading.)

**Comment by gwillen** [comment]
Hmmm, I gave myself a test reply from another account, and this time the notification icon got a small (1) next to it. I could _faintly believe_ I've never noticed it before just because it's a light grey on light grey next to light grey and is not very noticeable; should it have turned color or something? But I'm not aware of ever having clicked on the bell before, so it seems like if I had ever gotten a notification successfully _before_ now, I should have noticed the permanently-persistent number at some point... Also, the new notification had a different background color than the old ones; I _think_ I would have noticed that when I was checking before.

Also, I saw this error on console though I'm not sure exactly what I was doing when it happened:

"""
Uncaught TypeError: Cannot read property 'props' of undefined
    at r.handleTabTouchTap (VM2325 61e7d84ccd25bec5e229bd37ea270a8c38fbf758.js:9)
    at Object.u (VM2325 61e7d84ccd25bec5e229bd37ea270a8c38fbf758.js:9)
    at Object.invokeGuardedCallback (VM2325 61e7d84ccd25bec5e229bd37ea270a8c38fbf758.js:9)
    at Object.invokeGuardedCallbackAndCatchFirstError (VM2325 61e7d84ccd25bec5e229bd37ea270a8c38fbf758.js:9)
    at h (VM2325 61e7d84ccd25bec5e229bd37ea270a8c38fbf758.js:9)
    at g (VM2325 61e7d84ccd25bec5e229bd37ea270a8c38fbf758.js:9)
    at b (VM2325 61e7d84ccd25bec5e229bd37ea270a8c38fbf758.js:9)
    at Array.forEach (<anonymous>)
    at y (VM2325 61e7d84ccd25bec5e229bd37ea270a8c38fbf758.js:9)
    at j (VM2325 61e7d84ccd25bec5e229bd37ea270a8c38fbf758.js:9)
"""

**Comment by gwillen** [comment]
Hm, tried it again, this time the bell filled in and a 1 appeared next to it. Less subtle. The behavior seems inconsistent...

**Comment by gwillen** [comment]
Man, I dunno, maybe I'm losing my mind. It seems to be working fine now. I can't be sure the 'bug' is anything beyond 'the icon should probably light up in a brighter color so slow people like me actually notice it'. :-)

**Comment by gwillen** [comment]
Ok, now I opened the site and got a '15' on my notification-bell icon, which I'm sure I've never seen before, and this gives me a theory!

I think that notifications did not work for me _until_ the first time I clicked the bell icon, and after that they have worked normally.

@Raemon, is this an intentional effect somehow?

I have reproduced the effect with my gwillen-test account -- the notification bell icon does not light up or get a numbered badge when I reply to it. (I am currently refraining from testing that I can change this by clicking the icon, to maintain the account in the broken state for further inspection if desired.)
